### PR TITLE
Installer 0.7

### PR DESCRIPTION
**Pull request by modmuss50**
Currenly this PR contains:

- Removal of gson, decreasing the jar size

- General cleanup, File -> Path for example

- All network requests go through meta.fabricmc.net (The launcher and server libs are still downloaded from maven)

### CODE REVIEW

**Comment by modmuss50** [comment]
This should also fix: https://github.com/FabricMC/fabric-installer/issues/55

**Review by liach — APPROVED** [review]
Looks fantastic. The mjson lib is quite clean; I strongly support using it over Gson.

**Review by Mysterious-Dev — APPROVED** [review]

**Comment by modmuss50** [comment]
Should be ready for review/test.

File: build.gradle [hunk 1]
```diff
@@ -1,58 +1,63 @@
...
+	outputs.upToDateWhen { false }
+
+	doFirst {
+		output.delete()
+	}
+
+	doLast {
+		output.createNewFile()
+		output.setBytes downloadBootstrap.outputFiles.first().readBytes()
+		output.append shadowJar.archiveFile.get().getAsFile().readBytes()
```
Comment (liach) [inline build.gradle:57]: This looks really brute force :laughing:

**Review by liach — APPROVED** [review]

**Review by sfPlayer1 — APPROVED** [review]

**Pull request merged**